lib/matplotlib/testing/__init__.py

```diff
@@ -9,6 +9,8 @@
 import subprocess
 import sys
 
+import pytest
```

Comment:
burying this import in a function should fix the docs (pytest is not a build dependency, but we want to document things from this sub-package in the docs.